addons/mail/static/src/views/fields/properties_field/property_value.scss

```diff
@@ -0,0 +1,9 @@
+.o_property_field_value {
+    .o_property_many2one_avatar[comodel="res.users"],
+    .o_m2m_avatar[comodel="res.users"] {
+        &:hover {
+            cursor: pointer;
+            -webkit-filter: brightness(75%);
```

Comment:
We can maybe include the mixin `o-avatar-image-hover` instead of duplicating the rules ? It will ensure that the style will always be consistent.

Reply:
good idea :)